debian/control

```diff
@@ -50,6 +50,7 @@ Build-Depends: bc,
                python (>= 2.7),
                python-all-dev,
                python-nose,
+	       python-prettytable,
```

Comment:
Yeah, I think it needs to be here.  The question I have is whether the debbuilder stuff magically detects this python dependency as a package dependency too (the way it automatically figures out shared library dependencies) or whether the python dependency has to be explicitly declared...